utils/common/conststring.h

```diff
@@ -38,12 +38,31 @@ class ConstString
   explicit ConstString(const std::string& str) : mStr(str.data()), mLength(str.length())
   {
   }
+  template <typename T>
```

Comment:
Is it possible to calculate colWidth as sizeof(T) ?

Reply:
This is called from filtering part of column.cpp and that code keeps colWidth as a part of template argument. ConstStrings constructed there previously used COL_WIDTH instead of sizeof, I just factored that out.

Reply:
No.

This method gets called from column.cpp, with COL_WIDTH template parameter as argument. If COL_WIDTH is not removed from column.cpp's templates, I cannot remove it here.

This is "when in Rome do as Romans do" approach. I do not fully understand why COL_WIDTH is not computed as sizeof in column.cpp so I assume it is not possible and sizeof(T) is not always equal to COL_WIDTH.